examples/knowledge/cmd/read.go

```diff
@@ -32,15 +32,20 @@ var readCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 
-		path := "(n:Person)-[:OWNS]->(s:Store)"
-		conditions := "WHERE n.external_id = $external_id"
+		query := "MATCH (n:Person)-[r:CAN_SEE]->(a:Asset) WHERE n.id=$id"
 		params := map[string]*knowledgepb.InputParam{
 			"external_id": {
 				Value: &knowledgepb.InputParam_StringValue{StringValue: "1234"},
```

Comment:
in the query you say `WHERE n.id = $id` but in the input params you provide `external_id`. You need to commit to one 😄

Reply:
updated